broker-core/src/main/java/org/apache/qpid/server/security/group/FileGroupDatabase.java

```diff
@@ -104,17 +105,21 @@ public synchronized void setGroupFile(String groupFile) throws IOException
     @Override
     public synchronized void addUserToGroup(String user, String group)
     {
-        Set<String> users = _groupToUserMap.get(keySearch(_groupToUserMap.keySet(), group));
-        if (users == null)
+        Set<String> groupUsers = _groupToUserMap.get(keySearch(_groupToUserMap.keySet(), group));
```

Comment:
`keySearch(_groupToUserMap.keySet(), group)` is invoked multiple times in method.
Please, introduce a local variable to keep result of `keySearch(_groupToUserMap.keySet(), group)`

Reply:
Done!